contracts/base/LoansBase.sol

```diff
@@ -170,22 +164,93 @@ contract LoansBase is LoansInterface, Base {
         return lendingPool.cToken();
     }
 
-    /**
-        @notice Checks whether the loan's collateral ratio is considered to be secured based on the settings collateral buffer value.
-        @param loanID Id of the loan to check.
-        @return bool value of it being secured or not.
-    */
+    // See LoanLib.isSecured
     function isLoanSecured(uint256 loanID) external view returns (bool) {
-        return _isLoanSecured(loanID);
+        return loans[loanID].isSecured(settings());
     }
 
-    /**
-        @notice Checks whether a loan is allowed to be deposited to an Externally Owned Account.
-        @param loanID Id of the loan to check.
-        @return bool indicating whether the loan with specified parameters can be deposited to an EOA.
-     */
+    // See LoanLib.canGoToEOA
```

Comment:
I would add @dev See more details in LoanLib.canGoToEOA(...) function.
Same in other functions.

Reply:
Added